tardis/tests/tests_slow/conftest.py

```diff
@@ -0,0 +1,77 @@
+import os
+import numpy as np
+import pytest
+from astropy import units as u
+
+
+@pytest.fixture(scope="session")
+def slow_tests_datadir():
+    slow_tests_datadir = pytest.config.getvalue("slow-test-data")
+    if slow_tests_datadir is None:
+        pytest.skip('--slow-test-data was not specified')
+    else:
+        return os.path.expandvars(os.path.expanduser(slow_tests_datadir))
+
+
+@pytest.fixture(scope="session")
+def data_path():
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), "w7")
+
+
+@pytest.fixture(scope="session")
+def baseline(request, slow_tests_datadir):
```

Comment:
I don't like the naming of this fixture but if I'm the only one, don't bother changing it.
Something like `reference_data` would be far more descriptive in my opinion.

Same goes for `self.baseline`. Either call it `self.data` or `self.reference` or any combination of the two.